scripts/test/setup-integration-infra.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
```

Comment:
For all files intended to be run directly by Node.js, lets
- Introduce shebang, as here: https://github.com/serverless/serverless/blob/29d5fe75104c9bb0f44531e1c4b3491d4051f6b6/scripts/pkg/build.js#L1
- Ensure they're excutable (have executable permission bit)
- Have `essentials` loaded: https://github.com/serverless/serverless/blob/29d5fe75104c9bb0f44531e1c4b3491d4051f6b6/scripts/pkg/build.js#L7 as first thing